stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ui/MessagesActivity.kt

```diff
@@ -161,6 +161,9 @@ class MessagesActivity : BaseConnectedActivity() {
                     onUserAvatarClick = { user ->
                         Log.i("MessagesActivity", "user avatar clicked: ${user.id}")
                     },
+                    onTapUserMention = { user ->
```

Comment:
Hey @VIPlearner,

Thank you for the input!
Would you mind renaming `onTapUserMention` into `onUserMentionClick` for the sake of consistency throughout the PR?

Reply:
Hi @kanat done